repos/humanizer.proj

```diff
@@ -0,0 +1,48 @@
+<Project>
+
+  <PropertyGroup>
+    <SourceDirectory>Humanizer</SourceDirectory>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <PackagesOutput>$(ProjectDirectory)/src/Humanizer/bin/$(Configuration)</PackagesOutput>
+    <RepoApiImplemented>false</RepoApiImplemented>
+    <DeterministicBuildOptOut>true</DeterministicBuildOptOut>
+    <NuGetConfigFile>$(ProjectDirectory)/src/NuGet.config</NuGetConfigFile>
+  </PropertyGroup>
+
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+  <ItemGroup>
+    <!-- Humanizer needs SourceLink.Create.CommandLine which is not produced from sourcelink repo
+    <RepositoryReference Include="sourcelink" />
+    -->
+  </ItemGroup>
+
+  <Target Name="RepoBuild">
+    <PropertyGroup>
+      <BuildCommandArgs>$(ProjectDirectory)src/Humanizer/Humanizer.csproj</BuildCommandArgs>
+      <BuildCommandArgs>$(BuildCommandArgs) /p:Configuration=$(Configuration)</BuildCommandArgs>
+      <BuildCommandArgs>$(BuildCommandArgs) /v:$(LogVerbosity)</BuildCommandArgs>
+      <BuildCommandArgs>$(BuildCommandArgs) $(RedirectRepoOutputToLog)</BuildCommandArgs>
+    </PropertyGroup>
+
+    <Exec Command="$(DotnetToolCommand) restore /bl:restore.binlog $(BuildCommandArgs)  "
+          EnvironmentVariables="@(EnvironmentVariables)"
+          WorkingDirectory="$(ProjectDirectory)"
+          IgnoreStandardErrorWarningFormat="true" />
+
+    <Exec Command="$(DotnetToolCommand) build /bl:build.binlog $(BuildCommandArgs)"
+          EnvironmentVariables="@(EnvironmentVariables)"
+          WorkingDirectory="$(ProjectDirectory)"
+          IgnoreStandardErrorWarningFormat="true" />
+
+    <Exec Command="$(DotnetToolCommand) pack $(ProjectDirectory)src/Humanizer/Humanizer.csproj /p:NuspecFile=$(ProjectDirectory)NuSpecs/Humanizer.Core.nuspec /p:NuspecBasePath=$(ProjectDirectory)src/ /p:NuspecProperties=Version=2.2.0 /p:Configuration=$(Configuration)"
```

Comment:
Is there some way to pull the `Version` value from `eng/Version.Details.xml` and use it here instead of hardcoding `2.2.0`?

Reply:
You can add `<HumanizerCorePackageVersion>2.2.0</HumanizerCorePackageVersion>` to `eng/Versions.props` and use it from there.  If Humanizer.Core ever gets updated with auto-update, this value will update as well.

Reply:
Thanks! That would result in the version being in 2 places: `eng/Versions.props` and `eng/Version.Details.xml`. Is there some way to use the value from `Versions.props` in `Version.Details.xml`?

Reply:
No.  That's the pattern for ProdCon.  When these files are auto-updated, the version and sha get updated in Version.Details.xml and the corresponding version tag gets updated in Versions.props.